src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/ComposerWithSuggestions.js

```diff
@@ -220,26 +222,45 @@ function ComposerWithSuggestions({
                     debouncedUpdateFrequentlyUsedEmojis();
                 }
             }
-            const newCommentConverted = convertToLTRForComposer(newComment);
+
+            let newCommentConverted = newComment;
+            const prevComment = commentRef.current;
+
+            // This prevent the double execution of setting input value that could affect the place holder and could send an empty message or draft messages in android
+            if (prevComment !== newComment) {
+                newCommentConverted = removeUnicodeLTRWhenEmpty(newComment, newCommentConverted);
+                newCommentConverted = convertToLTRForComposer(newCommentConverted, composerIsEmpty);
```

Comment:
There are two methods being called in succession, with the variable being overwritten each time. It seems like these should be combined somehow (maybe using a parameter to clear the unicode character when empty).